NitroxPatcher/Patches/Persistent/GameSettings_SerializeInputSettings_Patch.cs

```diff
@@ -5,9 +5,9 @@
 using NitroxClient.MonoBehaviours.Gui.Input.KeyBindings;
 using NitroxModel.Helper;
 
-namespace NitroxPatcher.Patches.Dynamic
+namespace NitroxPatcher.Patches.Persistent
 {
```

Comment:
```suggestion
namespace NitroxPatcher.Patches.Persistent;
```

Reply:
I've applied this change in the new commits